electron/src/renderer/preload-app.ts

```diff
@@ -127,10 +127,10 @@ setupIpcInterface();
 subscribeToMainProcessEvents();
 
 window.addEventListener('DOMContentLoaded', addDragRegion);
-
 window.addEventListener('focus', () => {
   const selectedWebview = getSelectedWebview();
   if (selectedWebview) {
+    selectedWebview.blur();
```

Comment:
Is it the same trick used in `App.jsx`? If yes, let's put it in a utility to remember and reuse it.

Reply:
No, that one is different